.github/workflows/run_release.yaml

```diff
@@ -0,0 +1,59 @@
+on: 
+  workflow_dispatch:
+    inputs:
+      releaseName:
+        description: "Release Name"
+        required: true
+        type: string
+      version:
+        description: "Release Version"
+        required: true
+        type: string
+
+name: Build and Sign Release
+
+
+jobs:
+  build-and-deploy:
+    runs-on: windows-latest
+    env:
+      CODESIGN_PW: ${{ secrets.CODESIGN_PW }}
+      CODESIGN_PFX: ${{ secrets.CODESIGN_PFX }}
+    permissions:
+      contents: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: repo
```

Comment:
I created an extra level of directory structure to be doubly sure that the private key doesn't end up in the release artifact (even if the Remove-Item failed, say).

Before this is committed, I will look at eliminating that from the zip file when its created, but flagging for now as FYI.

Reply:
This is done now.